### PR TITLE
update main.php occurences to $baseurl

### PR DESCRIPTION
**Pull request by tedstrauss**

### CODE REVIEW

File: modules/README [hunk 1]
```diff
@@ -12,4 +12,4 @@ Modules should be entirely self-contained into a subdirectory with the following
         tools/        - This contains any helper tools used either manually or by cron for this module
 
 All of these directories are optional, and after a Menu_Filter is set up your module can
-be accessed in a standard Loris fashion at main.php?test_name=ModuleName[&subtest=...]
+be accessed in a standard Loris fashion at {$baseurl}/ModuleName[&subtest=...]
```
Comment (xlecours) [inline modules/README]: I think it is more like
`{$baseurl}/ModuleName/[subtestName/][?param1=value1&param2=value2]...`

At least, the slash after MonduleName is required

File: modules/instrument_list/js/instrument_list.js [hunk 2]
```diff
@@ -7,7 +7,7 @@ $(document).ready(function() {
     $(".instrument_list").click(function(e) {
         e.preventDefault();
         var form = $('<form />', {
-            "action" : "main.php?test_name=imaging_browser",
+            "action" : loris.BaseURL + "/imaging_browser",
```
Comment (xlecours) [inline modules/instrument_list/js/instrument_list.js]: missing trailing slash

Comment (tedstrauss) [inline modules/instrument_list/js/instrument_list.js]: so it should be like this?

loris.BaseURL + "/imaging_browser/",

**Comment by xlecours** [comment]
Code reviewed

File: modules/brainbrowser/templates/form_brainbrowser.tpl [hunk 3]
```diff
@@ -1,4 +1,4 @@
-<link rel="stylesheet" type="text/css" href="{$baseURL}/GetCSS.php?Module=brainbrowser&file=volume-viewer-demo.css">
+<link rel="stylesheet" type="text/css" href="/GetCSS.php?Module=brainbrowser&file=volume-viewer-demo.css">
```
Comment (jstirling91) [inline modules/brainbrowser/templates/form_brainbrowser.tpl]: why are you removing the base url?

**Pull request merged**